packages/Popover/src/Popover.js

```diff
@@ -26,6 +27,10 @@ const focusableElementSelector =
 // TODO: To handle cases where there are multiple scrolling containers, we need to implement
 //       getScrollContainer as oneOfType([func, arrayOf(func)])
 
+function isDescendantOfActiveElement() {
+  return isDescendant(document.activeElement, "data-pka-anchor", "popover.content");
```

Comment:
```
document.getElementById('overlay').contains("[data-pka-anchor='popover.content']")
```

assuming only one overlay exists per page

Reply:
I don't think it's safe to assume only one overly exists, but if it were you should be able to use this existing `helper` function:

https://github.com/acl-services/paprika/blob/master/packages/helpers/src/dom/isElementContainsFocus.js

For symmetry, maybe the helper you can add can be `isFocusContainedBy(isContainer)` 

where `isContainer = (el) => el.getAttribute("data-pka-anchor") === "popover.content"`

Reply:
in my previous comments, I pointed out exactly what you are pointed out to me lol...

Reply:
Yep, I'm aware of that and looking into it as mentioned in comment https://github.com/acl-services/paprika/pull/984#discussion_r599126881